### PR TITLE
Fix:-Replace the render method with the new createRoot method for fixture …

### PR DESCRIPTION
**Pull request by Biki-das**
Currently the Fixtures folder with the babel standalone folder which allows us to test our react changes is using the old method for the render, since react 18 is the current version, replacing it seems the way

Mozilla Firefox 2_16_2023 12_21_08 PM

### CODE REVIEW

File: fixtures/packaging/babel-standalone/dev.html [hunk 1]
```diff
@@ -5,10 +5,8 @@
...
-        <h1>Hello World!</h1>,
-        document.getElementById('container')
-      );
+      ReactDOM.createRoot( document.getElementById('container')).render(
+      <h1>Hello World!</h1>)
     </script>
   </body>
-</html>
\ No newline at end of file
+</html>;
```
Comment (eps1lon) [inline fixtures/packaging/babel-standalone/dev.html:12]: This semicolon looks wrong.

File: fixtures/packaging/babel-standalone/dev.html [hunk 2]
```diff
@@ -5,10 +5,8 @@
     <script src="https://unpkg.com/babel-standalone@6/babel.js"></script>
     <div id="container"></div>
     <script type="text/babel">
-      ReactDOM.render(
-        <h1>Hello World!</h1>,
-        document.getElementById('container')
-      );
+      ReactDOM.createRoot( document.getElementById('container')).render(
+      <h1>Hello World!</h1>)
```
Comment (eps1lon) [inline fixtures/packaging/babel-standalone/dev.html:9]:
```suggestion
      ReactDOM.createRoot(document.getElementById("container")).render(
        <h1>Hello World!</h1>
      );
```

File: fixtures/packaging/babel-standalone/dev.html (hunk 1 above)
Comment (Biki-das) [reply fixtures/packaging/babel-standalone/dev.html:12]: :-( fixing!!!!

File: fixtures/packaging/babel-standalone/dev.html (hunk 2 above)
Comment (eps1lon) [reply fixtures/packaging/babel-standalone/dev.html]: Why did the suggested formatting not work? It's in line with what Prettier outputs.

Comment (Biki-das) [reply fixtures/packaging/babel-standalone/dev.html]: seems like my prettier config tries to add colons to the end of the file! fixed it and ran prettier

Comment (Biki-das) [reply fixtures/packaging/babel-standalone/dev.html]: @eps1lon sorry for the wrong push! fixed the prettier error, this is ready to go now!

**Comment by Biki-das** [comment]
@eps1lon this is good to go i guess!

File: fixtures/packaging/babel-standalone/dev.html [hunk 3]
```diff
@@ -5,10 +5,8 @@
     <script src="https://unpkg.com/babel-standalone@6/babel.js"></script>
     <div id="container"></div>
     <script type="text/babel">
-      ReactDOM.render(
-        <h1>Hello World!</h1>,
-        document.getElementById('container')
-      );
+      ReactDOM.createRoot( document.getElementById('container')).render (
```
Comment (kushhingol) [inline fixtures/packaging/babel-standalone/dev.html:8]: I think the prettier configuration in our IDE is actually not helping to format and to remove whitespaces where ever required.

Suggesting the below changes
```
ReactDOM.createRoot(document.getElementById('container')).render(<h1>Hello World!</h1>);
```

**Comment by Biki-das** [comment]
@hoaxyq

**Pull request closed without merging**